config/manifests/bases/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -3,7 +3,7 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     alm-examples: '[]'
-    capabilities: Basic Install
+    capabilities: Full Lifecycle
```

Comment:
There is also the same field in `bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml`.
We might want to change it there now or it will be generated eventually thought.

Reply:
If I understand correctly, this field will change when the `make bundle` command executes. Should I do it now or wait for the release?